include/swift/SIL/SILFunction.h

```diff
@@ -900,6 +900,9 @@ class SILFunction
   /// block inside.  This depends on there being a 'dot' and 'gv' program in
   /// your path.
   void viewCFG() const;
```

Comment:
Instead of adding an additional method, how about just adding a default parameter `bool includingBBContents = true`?

Reply:
I think he is followed what LLVM does API wise. See: https://github.com/llvm-mirror/llvm/blob/c6da6867a1d033384367983944f7c1ad103e9559/include/llvm/IR/Function.h#L733.

The reason why I believe this is done is due to LLDB not supporting such sorts of default arguments. So you just get an error when calling viewCFG().

Reply:
No worries. Happy to help. = ).

Reply:
Indeed debuggers don't seem to support default arguments.